repo/content/content_manager_test.go

```diff
@@ -46,8 +46,7 @@ func TestMain(m *testing.M) { testutil.MyTestMain(m) }
 func TestContentManagerEmptyFlush(t *testing.T) {
 	ctx := testlogging.Context(t)
 	data := blobtesting.DataMap{}
-	keyTime := map[blob.ID]time.Time{}
-	bm := newTestContentManager(t, data, keyTime, nil)
+	bm := newTestContentManager(t, data)
```

Comment:
it seems that this test refactoring could be (or have been) merged on its own upfront.

Reply:
done